sklearn/gaussian_process/tests/test_gaussian_process.py

```diff
@@ -129,8 +129,11 @@ def test_ordinary_kriging():
 
 def test_no_normalize():
     gp = GaussianProcess(normalize=False).fit(X, y)
-    y_pred = gp.predict(X)
+    y_pred = gp.predict(X, batch_size=1)
     assert_true(np.allclose(y_pred, y))
+    y_pred = gp.predict(X, eval_MSE=True)
```

Comment:
As suggested by @lesteve, this doesn't belong in `test_no_normalize`. Make a new test for `batch_size` only.